plugins/Microsoft.DocAsCode.Build.UniversalReference/BuildOutputs/ApiInheritanceTreeBuildOutput.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Microsoft.DocAsCode.Build.UniversalReference.BuildOutputs
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Microsoft.DocAsCode.DataContracts.Common;
+
+    using Newtonsoft.Json;
+    using YamlDotNet.Serialization;
+
+    [Serializable]
+    public class ApiInheritanceTreeBuildOutput
```

Comment:
yes, should have

---
In reply to: [113361121](https://github.com/dotnet/docfx/pull/1596#discussion_r113361121) [](ancestors = 113361121)